src/esp/gfx/replay/Keyframe.h

```diff
@@ -55,6 +55,7 @@ struct Keyframe {
   std::vector<std::pair<RenderAssetInstanceKey, RenderAssetInstanceState>>
       stateUpdates;
   std::unordered_map<std::string, Transform> userTransforms;
+  Cr::Containers::Optional<std::vector<LightInfo>> lights;
```

Comment:
That's correct.

I picked this format because it's simple, and because we rarely change lights during a simulation. Therefore, eliminating duplicates from consecutive light set changes won't optimize the replay file size in most cases.

I'll add a comment to clarify how we use the data here.